yorkie/src/main/kotlin/dev/yorkie/util/IndexTree.kt

```diff
@@ -397,19 +398,31 @@ internal abstract class IndexTreeNode<T : IndexTreeNode<T>>(children: MutableLis
             return parent?.children?.getOrNull(offset + 1)
         }
 
+    private var canUseCachedChildren = false
```

Comment:
I think it's a good approach to reduce performance overhead, but I'm a bit concerned that updating `canUserCachedChildren` might be overlooked when implementing new methods that update `childrenInternal` later. Adding a comment on this might be helpful, maybe...🤔

Reply:
Implemented IndexTreeNodeList to handle this better [6c93e13](https://github.com/yorkie-team/yorkie-android-sdk/pull/174/commits/6c93e130f20ea991186932a2487a86b326fbe009)